drv.go

```diff
@@ -356,6 +356,13 @@ func (d *drv) acquireConn(pool *connPool, P commonAndConnParams) (*C.dpiConn, bo
 				cs := C.CString(value)
 				defer C.free(unsafe.Pointer(cs))
 				C.dpiData_setBytes(&tempData, cs, C.uint32_t(len(value)))
+			case []byte:
+				columns[i].oracleTypeNum = C.DPI_ORACLE_TYPE_RAW
+				columns[i].nativeTypeNum = C.DPI_NATIVE_TYPE_BYTES
+				bytesVal := value.([]byte)
+				cs := (*C.char)(C.CBytes(bytesVal))
```

Comment:
AFAIK tempData won't hold on to our pointer, so this copying is unneeded:
`C.dpiData_setBytes(&tempData, (*C.char)(unsafe.Pointer(&value[0])), len(value))`
is enough

Reply:
Will test that. Thanks.

Reply:
Works fine. Thanks.